Lib/bdb.py

```diff
@@ -472,6 +632,7 @@ def runcall(self, func, *args, **kwds):
 
 
 def set_trace():
+    """Start debugging with a Bdb instance from the caller's frame."""
```

Comment:
Yes, it's from the docs.  There are three functions defined outside of the classes - set_trace(), effective(), and checkfuncname().  effective() and checkfuncname() already had docstrings, so I didn't change them.  set_trace() has one line - Bdb.set_trace().